contrib/hack/test_walkthrough.sh

```diff
@@ -114,6 +115,14 @@ retry -n 10 \
 
 echo 'Deploying service catalog...'
 
+# The API server automatically provisions the configmap, but we need it to contain the requestheader CA as well,
+# which is only provisioned if you pass the appropriate flags to master.  GKE doesn't do this, so we work around it.
```

Comment:
anything we need to do specially in GKE in the walk-through when 1.7 comes around?